components/mpas-ocean/src/Registry.xml

```diff
@@ -202,7 +202,7 @@
 		/>
 		<nml_option name="config_time_integrator" type="character" default_value="split_explicit_ab2"
 					description="Time integration method."
-					possible_values="'split_explicit', 'RK4', 'unsplit_explicit', 'split_implicit', 'LTS', 'split_explicit_ab2'"
+					possible_values="'split_explicit', 'RK4', 'unsplit_explicit', 'split_implicit', 'LTS', 'FB_LTS', 'split_explicit_ab2'"
```

Comment:
I think we need a comment in these possible_values about which are useable by E3SM and which are currently limited to the standalone model. The Registry file gets used to make bld files in E3SM and this having these options there could cause confusion

Reply:
Good point. I added that information to the description, because I didn't want to add text to the list in `possible_values`.